Readme.md

```diff
@@ -51,6 +51,7 @@ extension=/opt/bref-extra/amqp.so
 | Mailparse | `${bref:extra.mailparse-php-74}` | `extension=/opt/bref-extra/mailparse.so` |
 | Memcache | `${bref:extra.memcached-php-74}` | `extension=/opt/bref-extra/memcache.so` |
 | Memcached | `${bref:extra.memcached-php-74}` | `extension=/opt/bref-extra/memcached.so` |
+| ODBC | NaN | `extension=/opt/bref-extra/odbc.so` |
```

Comment:
Let's rename this to Snowflake and I'll be happy ro merge. 
```suggestion
| Snowflake | `${bref:extra.snowflake-php-74}` | `extension=/opt/bref-extra/snowflake.so` |
```

Reply:
The odbc extension itself is the same for any future odbc layer and you can't load the same extension twice. I would prefer keeping the file `odbc.so`
Also, Snowflake is working on a pdo extension that just isn't ready yet so that would be a better candidate for the "snowflake" name.

How about naming it `odbc-snowflake` and continue calling the file `odbc.so`? Then if there would be another odbc layer, you can simply include the one odbc extension and configure both libraries though an own ini file.